README.md

```diff
@@ -1,104 +1,151 @@
+----
 
-## User Story
+CentOS Community Container Pipeline (CCCP) is an open-source platform to containerize applications based on CentOS.
 
-As an application developer, I want to develop applications on a stack (django, golang, nodejs, redis, rabbitmq, etc.) of my choice using CentOS as the base platform. I want to make sure that the application is packaged into a container and updated  automatically every time I push changes to my Git (GitHub, BitBucket, Gitlab, etc.) repository; the resulting container image is scanned for updates, fixes, capabilities, and delivered to a public registry from where the users can pull the image and run the application. I also want the container image to be automatically rebuilt when an RPM package is updated in the repository or base image (`FROM` in Dockerfile) is updated.
+**CCCP:** Builds the application (from a git repository) → Packages with the appropriate runtime → Tests and analyzes the image → Deploys to a public registry
 
-## How does it work?
+----
 
-A developer working on an open-source project opens a pull request (PR) on [Container Index](https://github.com/CentOS/container-index/) to use Container Pipeline Service for building container images. Once the PR is merged, Container Pipeline Service lints the Dockerfile, builds the image for his/her project, scans it, pushes it to registry.centos.org (Web UI coming soon!), and finally notifies the developer via email.
+## Use Case
 
-Once a project is in the Container Index, the Container Pipeline Service service automatically tracks project's Git repo + branch for changes and rebuilds it every time there is a change.
+I have a certain stack I develop with (be it Django, Golang, NodeJS, Redis, RabbitMQ, etc.) using my favourite OS (CentOS) as a base platform. 
 
-**NOTE:** It might take some time for the build to finish as it depends on the number of jobs in the queue. If it's taking long, [contact us](#get-in-touch).
+How do I package that application into a container that's updated automatically every time I push changes? What about security and updates, how do I automate that each time I push any changes?
 
-The entire flow can be summarized as below
+That's where CCCP comes in. 
 
-![Container Pipeline Diagram](docs/diagrams/architecture.png)
+CCCP will:
+
+  - Scan the image for updates, fixes, capabilities and push it to a public registry (by default, http://registry.centos.org)
+  - Automatically rebuild when an RPM package is updated in the repository or base image (`FROM` in Dockerfile) is updated
```

Comment:
Lets add 
* image update based on every push in source repo.
* also a line about weekly scan emails for notifying the user about scan results (here we do not update the image, we just notify the user about scan results)